### PR TITLE
moving snowflake from consumers to producers

### PR DESCRIPTION
**Pull request by Sheeri**
Here's a screenshot of the result on my local build:

Screenshot 2024-07-02 at 2 37 50 PM

### CODE REVIEW

**Comment by Sheeri** [comment]
cc @julienledem can you give approval that Snowflake should be listed as a producer, not a consumer?

File: static/ecosystem/consumers.tsx [hunk 1]
```diff
@@ -87,12 +87,5 @@ export const Consumers: Array<Partner> = [
...
     org_url: "https://open-metadata.org/"
-  },
-  {
-    image: "snowflake_logo.svg",
-    org: "Snowflake",
-    description: "Snowflake's OpenLineage Adapter creates an account-scoped view from ACCESS_HISTORY and QUERY_HISTORY to output each query that accesses tables in OpenLineage JsonSchema specification.",
-    docs_url: "https://github.com/Snowflake-Labs/OpenLineage-AccessHistory-Setup",
-    org_url: "https://developers.snowflake.com/"
-  }
+  },
```
Comment (julienledem) [inline static/ecosystem/consumers.tsx:90]: Not sure about the comma here. But if it works on your local build then it's fine.

**Review by julienledem — APPROVED** [review]
Thank you Sheeri

File: static/ecosystem/consumers.tsx (hunk 1 above)
Comment (julienledem) [reply static/ecosystem/consumers.tsx:90]: the deploy-preview confirms that this works. Resolving

**Pull request merged**